extensions/scarb-cairo-test/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "scarb-cairo-test"
+version.workspace = true
+rust-version.workspace = true
+edition.workspace = true
+publish = false
+
+authors.workspace = true
+
+[dependencies]
+anyhow.workspace = true
+cairo-lang-compiler = { version = "1.1.0-rc0", git = "https://github.com/starkware-libs/cairo.git", rev = "9d6cd56" }
```

Comment:
Why not `1.1.0` instead? 🤔

Reply:
I would have to use even newer commit here, because at that particular one the version haven't been changed yet :(

Reply:
And this commit if "after" 1.1.0 release